templates/components/common/forms/date.html

```diff
@@ -37,6 +38,7 @@
                     data-label="year"
                     data-input
                     aria-required="{{required}}"
+                    aria-labelledby="{{id}}"
                     {{#if private_id}}data-field-type="{{private_id}}"{{/if}}
```

Comment:
do we need to add the `aria-live` and `aria-labelledby` attributes to the day selection as well?

Reply:
Good call